src/pcap/.golangci.yml

```diff
@@ -286,3 +285,6 @@ issues:
     - gosec
     - noctx
     - wrapcheck
+#  - path: "\\.pb\\.go"
+#    linters: []
+
```

Comment:
Seems like something sneaked in here that shouldn't.

Reply:
yes it did. fixed now.